tests/io_test.py

```diff
@@ -501,7 +501,7 @@ def testGetDataSets(self):
     def testSaveStringsDataset(self):
         # Create, save, and load Strings dataset
         strings_array = ak.array(["testing string{}".format(num) for num in list(range(0, 25))])
-        strings_array.save("{}/strings-test".format(IOTest.io_test_dir), dataset="strings")
+        strings_array.to_hdf("{}/strings-test".format(IOTest.io_test_dir), dataset="strings")
```

Comment:
Just adding one comment for all the tests here - `to_parquet` versions?

Reply:
Again, there is a specific file for testing Parquet.